bigquery_etl/metadata/parse_metadata.py

```diff
@@ -42,8 +72,10 @@ def is_metadata_file(file_path):
 
         Checks if the name and file format match the metadata file requirements.
         """
-        # todo: we should probably also check if the file actually exists etc.
-        return os.path.basename(file_path) == METADATA_FILE
```

Comment:
I updated this method to include file checking per the TODO, but CircleCI failed it. Should I revert this change or is there a way to update the CircleCI test?